src/Symfony/Component/HttpFoundation/Session/Storage/Handler/StrictSessionHandler.php

```diff
@@ -30,6 +30,16 @@ public function __construct(\SessionHandlerInterface $handler)
         $this->handler = $handler;
     }
 
+    /**
+     * Returns true if this handler wraps an internal PHP session save handler using \SessionHandler.
+     *
+     * @internal This method is public to avoid a bug inside the `SessionHandlerProxy` class. It should not be used in user code.
```

Comment:
Let's remove the note after `@internal`, it doesn't bring anything to me.
Let's also remove the `final` keyword, `@internal` is enough.